README.md

```diff
@@ -1 +1,58 @@
-# HTML-CSS-JavaScript-capstone-project
\ No newline at end of file
+ Description of the project: 
+    This project represents the mobile version and the Desktop version of the first capstone project from Microverse, Where I used all the skills and knowledge that I acquired form the first module.
+
+
+## Built With
+
+- Major languages: HTML
+-  CSS
+    - Flexbox
+    - CSS Grid
+- JavaScript
+- Mobile first aproach
+- linter for code analysis
+
+## Deployment
+https://mohammedelseddik.github.io/Web-Development-Capstone-Project/
+
+
+## Screenshot
+* Mobile version
+![Mobile version](https://user-images.githubusercontent.com/97116942/157699201-fbf3cd82-25ac-4541-96e9-6035768e7dd7.PNG)
+
+
+* Desktop version
+![Desktop version](https://user-images.githubusercontent.com/97116942/157699325-9f44d40f-e533-4fb3-8f14-804b5710b733.PNG)
+
+
+## Authors
+
+👤 **Mohammed Elseddik Rebbouh**
+
+- GitHub: [@githubhandle](https://github.com/MohammedElseddik)
+- Twitter: [@twitterhandle](https://twitter.com/RebbouhElseddik)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammed-elseddik-rebbouh-676500192)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
Here we need to have a link to the issues page of our project. In the case of your project, the link is this one:
[issues page](https://github.com/MohammedElseddik/HTML-CSS-JavaScript-capstone-project/issues)

- [ ] Please take care of this aspect, implementing the right link for issues.